pyproject.toml

```diff
@@ -8,7 +8,7 @@ requires-python = ">=3.7"
 dynamic = ["version"]
 name = "pip-tools"
 description = "pip-tools keeps your pinned dependencies fresh."
-readme = "README.rst"
+readme = "README.md"
```

Comment:
I wonder if this will need a long description type set explicitly...

Reply:
Try building the dist via `python -m dist` (there might be a tox env for this) and inspect it with `twine --strict`.

Reply:
> I wonder if this will need a long description type set explicitly...

I changed this line because I received a traceback about missing long description.

Reply:
I know. I'm just not sure if that should be paired with another setting.

Reply:
Look at the table @ https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata:
It says:
> If `content-type` is not given, `"text/x-rst"` is used by default.

So I'm almost sure this would render poorly on PyPI. Could you build the package off of this branch, then unpack the tarball (`python -m tarfile`) and the wheel (`wheel unpack`) and look into the metadata files to see if there's a mention of the markdown type? If it doesn't say markdown, maybe we have to set that explicitly.

Reply:
The METADATA file of the wheel and the PKG-INFO of the built package contain the following line:

```
Description-Content-Type: text/markdown
```

This seems to point out that explicitly specifying the content type in the pyproject.toml is not necessary. I would also be surprised about everything else, since the [python packaging guide](https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata) recommends this configuration.

Reply:
Marking the thread as resolved.